.github/workflows/weaver_deploy_go-pkgs.yml

```diff
@@ -371,7 +371,7 @@ jobs:
         working-directory: weaver/sdks/fabric/go-sdk
         
       - name: Build test
-        run: go build -v ./...
+        run: make build
```

Comment:
Can you replace the actions in the `build` target in the `Makefile` with `go build -v ./...`? I know that only some of the subfolders have unit tests configured, so you can keep the `test` target as is, but we do want to ensure that the code in all the subfolders are building fine.

Reply:
Done. This involved fixing the build of go-sdk, so I've changed PR name to fix, as there was a bug.